includes/admin/settings/register-settings.php

```diff
@@ -898,6 +898,12 @@ function edd_get_registered_settings() {
 		)
 	);
 
+	if ( ! edd_shop_supports_buy_now() ) {
+		$edd_settings['misc']['button_text']['buy_now_text']['disabled']      = true;
```

Comment:
@pippinsplugins we're doing both. This way, if they look at the item itself they see it grayed out (with updated tooltip), and if they go to the button text b/c they think the text isn't set correctly, they see the reason why it's not changing.